src/mame/heathkit/intr_cntrl.cpp

```diff
@@ -71,24 +91,7 @@ uint8_t heath_intr_cntrl::get_instruction()
 
 		return 0x00;
 	}
-
-	uint8_t level = 0;
-	uint8_t mask = 0x01;
-
-	while (mask)
-	{
-		if (m_intr_lines & mask)
-		{
-			break;
-		}
-		level++;
-		mask <<= 1;
-	}
-
-	if (level > 7)
-	{
-		logerror("bad level: %d\n", level);
-	}
+	uint8_t level = ffs(m_intr_lines) - 1;
```

Comment:
`ffs` isn’t available on Windows.  You’ll need the loop for now.

Reply:
That's disappointing, builds showed it was fine with Windows and CLANG, just not Windows and GCC.

Reply:
That’s only because the Windows clang build only includes a minimal set of systems.  It’s mostly for testing that the Windows-specific code in src/osd can be compiled with clang (the rest of MAME is built with clang in Linux and macOS CI jobs).  This code would still fail to build with MinGW clang on Windows.